src/Compilers/CSharp/Portable/Syntax/BaseExpressionColonSyntax.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax
+{
+    partial class BaseExpressionColonSyntax
+    {
+        public IdentifierNameSyntax? Name => Expression as IdentifierNameSyntax;
```

Comment:
>Name [](http://example.com/codeflow?start=37&length=4)

Where is this property used?

Reply:
(meant to be for backcompat scenario, but now I see this is not needed)